code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -558,7 +558,7 @@
 			to_chat(usr, span_warning("This unit already has an expand module installed!"))
 			return FALSE
 		// SKYRAT EDIT BEGIN
-		if(R_TRAIT_WIDE in R.model.model_features)
+		if(R.can_rest()) //	This proc returns TRUE if the robot has TALL or WIDE traits
```

Comment:
This is a bad way to do this, it should just check for wide or tall specifically because if can_rest changes then this proc breaks.